CICD/CICD.Release.Preview.cs

```diff
@@ -77,6 +77,14 @@ public partial class CICD // Release.Preview
                     $"{Environment.NewLine}{ConsoleTab}To view the milestone, go here 👉🏼 {updatedMilestone.HtmlUrl}{Environment.NewLine}";
                 Log.Information(updateMsg);
 
+                // If README pre-processing should be performed, process readme before packaging
```

Comment:
I know these are just comments, but you need to make a decision on README or readme or Readme.  I say README is more prominent, so I would refactor to README.
```suggestion
                // If README pre-processing should be performed, process README before packaging
```